Author: Ryan Niebur <ryanryan52@gmail.com>
Description: by default --avoid-ldap since apache2 is the only user, and we
Index: apr-util-1.3.12/apu-config.in
===================================================================
--- apr-util-1.3.12.orig/apu-config.in	2010-09-29 18:22:45.000000000 +0400
+++ apr-util-1.3.12/apu-config.in	2011-05-24 16:22:36.302660081 +0400
@@ -30,7 +30,8 @@
 LIBS="@APRUTIL_EXPORT_LIBS@"
 INCLUDES="@APRUTIL_INCLUDES@"
 LDFLAGS="@APRUTIL_LDFLAGS@"
-LDAP_LIBS="@LDADD_ldap@"
+ORIG_LDAP_LIBS="@LDADD_ldap@"
+LDAP_LIBS=""
 DBM_LIBS="@LDADD_dbm_db@ @LDADD_dbm_gdbm@ @LDADD_dbm_ndbm@"
 
 APRUTIL_LIBNAME="@APRUTIL_LIBNAME@"
@@ -55,7 +56,7 @@
   --includedir      print location where headers are installed
   --ldflags         print linker flags
   --libs            print library information
-  --avoid-ldap      do not include ldap library information with --libs
+  --avoid-ldap      do not include ldap library information with --libs (default on Debian)
   --ldap-libs       print additional library information to link with ldap
   --avoid-dbm       do not include DBM library information with --libs
   --dbm-libs        print additional library information to link with DBM
@@ -125,7 +126,7 @@
     flags="$flags $LDAP_LIBS $DBM_LIBS $LIBS"
     ;;
     --ldap-libs)
-    flags="$flags $LDAP_LIBS"
+    flags="$flags $ORIG_LDAP_LIBS"
     ;;
     --dbm-libs)
     flags="$flags $DBM_LIBS"
